hazelcast-sql/pom.xml

```diff
@@ -38,7 +38,7 @@
 
         <checkstyle.headerLocation>${project.parent.basedir}/checkstyle/ClassHeaderHazelcastCommunity.txt</checkstyle.headerLocation>
 
-        <confluent.version>6.2.2</confluent.version>
+        <confluent.version>7.4.0</confluent.version>
```

Comment:
Could you change com.hazelcast.jet.kafka.impl.DockerizedKafkaTestSupport#TEST_KAFKA_VERSION as well?